migrations/1_initial_migration.js

```diff
@@ -1,7 +1,5 @@
-const Migrations = artifacts.require('./Migrations.sol');
 const Unipool = artifacts.require('./Unipool.sol');
 
 module.exports = function (deployer) {
-    deployer.deploy(Migrations);
```

Comment:
Do we still need this file and the Migrations contract?

Reply:
I was using `truffle migrate` to deploy.